src/axom/inlet/inlet_utils.hpp

```diff
@@ -0,0 +1,16 @@
+#include "axom/sidre.hpp"
+
+#ifndef INLET_UTILS_HPP
+#define INLET_UTILS_HPP
+
+namespace axom
+{
+namespace inlet
+{
+
+void setWarningFlag(axom::sidre::Group* root);

```

Comment:
Document this function please.. just a quick "This function is used to mark if anything went wrong during the defining phase of inlet so verify() will properly fail."